src/multi.rs

```diff
@@ -341,10 +341,6 @@ impl Multi {
     /// error code from the transfer or just the fact that a transfer is
     /// completed. More details on these should be written down as well.
     pub fn messages<F>(&self, mut f: F) where F: FnMut(Message) {
-        self._messages(&mut f)
-    }
-
-    fn _messages(&self, mut f: &mut FnMut(Message)) {
```

Comment:
Oh could this be left as-is? The intention here is to reduce monomorphization burden on callers

Reply:
Ah I see, TIL. Removed the second commit.